packages/worker/src/worker.ts

```diff
@@ -149,15 +163,26 @@ export class NativeWorker implements NativeWorkerLike {
   public readonly recordActivityHeartbeat: OmitFirstParam<typeof native.workerRecordActivityHeartbeat>;
   public readonly initiateShutdown: Promisify<OmitFirstParam<typeof native.workerInitiateShutdown>>;
 
-  public static async create(connection: NativeConnection, options: CompiledWorkerOptions): Promise<NativeWorkerLike> {
+  public static async create(
+    connection: NativeConnection,
+    options: CompiledWorkerOptions,
+    bundle?: WorkflowBundleWithSourceMap
+  ): Promise<NativeWorkerLike> {
     const runtime = Runtime.instance();
-    const nativeWorker = await runtime.registerWorker(extractNativeClient(connection), options);
+    const nativeWorker = await runtime.registerWorker(
+      extractNativeClient(connection),
+      addBuildId(options, bundle?.code)
```

Comment:
I don't think we should automatically attach the autogenerated build id no matter what.

Users should be able to override it if they so desire but it looks like that's not possible if they pass a bundle right now.

Reply:
Yeah, an oversight, thanks.